examples/Example13/example13.cu

```diff
@@ -120,7 +120,13 @@ __global__ void InitPrimaries(ParticleGenerator generator, int startEvent, int n
     track.initialRange = -1.0;
 
     track.pos = {0, 0, 0};
-    track.dir = {1.0, 0, 0};
+    if (rotatingParticleGun) {
+      const double phi = 2. * 3.14159 * i / numEvents;
+      const double eta = -5. + 10. * i / numEvents;
+      track.dir = {cos(phi), sin(phi), sinh(eta)};
```

Comment:
Is this normalized anywhere?

Reply:
I was wondering if it has to be normalised. I interpreted this as a direction. Momentum should follow from the energy and the mass, so it shouldn't matter what the normalisation is. So tell me: Do I have to normalise (and why if you happen to know)?

Reply:
Not normalizing direction generates havoc for geometry since all propagations are made like: `point +=  dir*distance` assuming dir is a unit vector. Also, if you need the momentum components (used in field propagation) they will be wrong. So yes, normalization has to be done.

Reply:
Confirmed (and now fixed):
```
Normalised                                                    | Normalised * 5.

Simulating particles 1 ... done!				Simulating particles 1 ... done!
Run time: 112.829					      |	Run time: 99.9913


Mean energy deposit          8.13637 GeV		      |	Mean energy deposit          8.12393 GeV
Mean number of gamma         3506.79			      |	Mean number of gamma         3496.55
Mean number of e-            8167.55			      |	Mean number of e-            8205.67
Mean number of e+            359.98			      |	Mean number of e+            357.56
Mean number of charged steps 21586.8			      |	Mean number of charged steps 20658.6
Mean number of neutral steps 30942.1			      |	Mean number of neutral steps 27520.2
Mean number of hits          27120.3			      |	Mean number of hits          23151.3
Killed in propagation:       867			      |	Killed in propagation:       740
```